cheroot/connections.py

```diff
@@ -16,7 +15,7 @@
 try:
     import fcntl
 except ImportError:
-    try:
+    try:  # noqa: WPS505
```

Comment:
If we're starting to move suppressions inline, we should do so in a separate PR. Plus include explanations.